architecture/core_concepts/routes.adoc

```diff
@@ -37,6 +43,29 @@ Configuration] guide for information on deploying a router.
 ====
 endif::[]
 
+A router uses the service selector to find the service and the endpoints backing
+the service. Service-provided load balancing is bypassed and replaced with the
+router's own load balancing. Routers watch the cluster API and automatically
+update their own configuration according to any relevant changes in the API
+objects. Routers may be containerized or virtual. Custom routers can be deployed
+to communicate modifications of API objects to an external routing solution.
+
+In order to reach a router in the first place, requests for host names must
+resolve via DNS to a router or set of routers. The suggested method is to define
+a cloud domain with a wildcard DNS entry pointing to a virtual IP (VIP) backed by
+multiple router instances on designated nodes. Routes can be sharded among the set
```

Comment:
This is the overview of routing so I think its relevant. I can point to the section and introduce it better.